### ISSUE TITLE
When creating a new record in an entity, it is not automatically selected

### ISSUE THREAD

**Issue by marieke-bijlsma** [molgenis/molgenis#5027]
#### Reproduce
1. Upload this file: https://drive.google.com/open?id=0B8iNw4kI1MA2VHRhcl84cU1Ec0E
2. Go to entity 'cohorts' in data explorer
3. Click on '+' button
4. Click on '+' button in the field 'Imaging data' and create a random new 'Imaging data' record (Dicom image can be left empty)

#### Expected
The newly created 'Imaging data' record would be automatically selected.

#### Observed
Nothing selected.

**Comment by Mark-de-Haan** [comment]
Not really a bug, more a feature request. Currently implementing this would require a relatively big refactoring of how we generate our forms.

Propose to add label wont-fix

**Comment by Mark-de-Haan** [comment]
Wontfix: Issue will be adressed in the [VueForms](https://github.com/Mark-de-Haan/molgenis-vue-forms)

**Comment by mswertz** [comment]
I don't think it will be addressed when I look at current state of these forms

**Comment by mswertz** [comment]
very often required feature
